src/components/star-rating.less

```diff
@@ -0,0 +1,20 @@
+.StarRating {
+  display: inline-grid;
+  align-items: baseline;
+  justify-content: start;
+  grid-auto-flow: column;
+}
+
+.StarRating-value {
+  padding-right: 10px;
+}
+
+.StarRating-rating {
+  .ant-rate-star {
+    margin-inline-end: 0 !important;
```

Comment:
Unfortunately, the AntD rate component has everything hard-coded (including the star colors), so we cannot easily customize. But IMO the stars had too much space in between by default (8px), so I removed the space here forcefully.